### ISSUE TITLE
Content of PWAs doesn't load or looks distorted on macOS

### ISSUE THREAD

**Issue by mathiasba** [filips123/PWAsForFirefox#588]
### Description

Since several weeks the content of some PWAs (e. g. Google Keep) won't load anymore. Other PWAs (e. g. YouTube Music) load some content, but it's completely distorted.

I made sure that the native version 2.15.5 is istalled, but it still doesn't work for me. "Patch profiles and runtime" also didn't change anything, unfortunately.

### Screenshots

<img width="1392" alt="Bildschirmfoto 2024-11-12 um 11 23 33" src="https://github.com/user-attachments/assets/25c8b338-3e7f-4bcf-bbcd-de7180ffb361">

<img width="1435" alt="Bildschirmfoto 2024-11-12 um 11 24 05" src="https://github.com/user-attachments/assets/16af3c52-6115-4c45-b7ba-0161d76717ce">

### Environment

* Operating system: macOS Sonoma 14.6.1
* System architecture: ARM (Apple M2)
* Installation method: AMO (Firefox Add-On Store)
* PWAsForFirefox extension version: 2.12.5
* PWAsForFirefox native version: 2.12.5
* PWAsForFirefox runtime version: 131.0.3
* Firefox version: 132.0.1 (64-Bit), but the error also occurred with version 131.0.3

**Comment by filips123** [comment]
What if you install a new PWA into a fresh profile? Also, please check and provide [the runtime logs](https://pwasforfirefox.filips.si/help/troubleshooting/#obtaining-runtime-logs).

**Comment by mathiasba** [comment]
I already installed Google Keep in a new profile before writing the Issue. After doing it with YouTube Music as well, it now shows the same behavior as Google Keep (no content at all).

After launching the web app from the command line, I got the following error messages:

<img width="1457" alt="grafik" src="https://github.com/user-attachments/assets/dfe5a3fc-3589-47ba-bafa-1d27a4bfd008">


`TypeError: NetworkError: Network request failed
    onerror resource://services-settings/Utils.sys.mjs:238
RemoteSettingsClient.sys.mjs:533:15
TypeError: NetworkError: Network request failed
    onerror resource://services-settings/Utils.sys.mjs:238
RemoteSettingsClient.sys.mjs:533:15
Region.sys.mjs: Error fetching region TypeError: NetworkError when attempting to fetch resource. Region.sys.mjs:409:11
Region.sys.mjs: Failed to fetch region Error: NO_RESULT
    _getRegion resource://gre/modules/Region.sys.mjs:411
Region.sys.mjs:208:11
TypeError: currentPlacements is undefined
CustomizableUI.sys.mjs:3726:11
TypeError: currentPlacements is undefined
CustomizableUI.sys.mjs:3726:11
TopSitesFeed: Failed to fetch data from Contile server: NetworkError when attempting to fetch resource. TopSitesFeed.sys.mjs:542
TypeError: NetworkError: Network request failed
    onerror resource://services-settings/Utils.sys.mjs:238
RemoteSettingsClient.sys.mjs:533:15
TypeError: NetworkError: Network request failed
    onerror resource://services-settings/Utils.sys.mjs:238
IgnoreLists.sys.mjs:87:15
TypeError: NetworkError: Network request failed
    onerror resource://services-settings/Utils.sys.mjs:238
RemoteSettingsClient.sys.mjs:533:15
TypeError: NetworkError: Network request failed
    onerror resource://services-settings/Utils.sys.mjs:238
RemoteSettingsExperimentLoader.sys.mjs:312:15
TypeError: NetworkError: Network request failed
    onerror resource://services-settings/Utils.sys.mjs:238
RemoteSettingsClient.sys.mjs:533:15
TypeError: NetworkError: Network request failed
    onerror resource://services-settings/Utils.sys.mjs:238
RemoteSettingsExperimentLoader.sys.mjs:312:15
TypeError: NetworkError: Network request failed
    onerror resource://services-settings/Utils.sys.mjs:238
RemoteSettingsClient.sys.mjs:533:15
TypeError: NetworkError: Network request failed
    onerror resource://services-settings/Utils.sys.mjs:238
RemoteSettingsClient.sys.mjs:533:15
ASRouter: CheckBrowserNeedsUpdate failed :>>  
XMLHttpRequest { readyState: 4, timeout: 0, withCredentials: false, upload: XMLHttpRequestUpload, responseURL: "https://aus5.mozilla.org/update/6/Firefox/131.0.3/20241011205646/Darwin_aarch64-gcc3/en-US/release/Darwin%252023.6.0/ISET%3ANEON%2CMEM%3A16384/firefoxpwa/2.13.1/update.xml?force=1", status: 0, statusText: "", responseType: "", response: "", responseText: "" }
ASRouterTargeting.sys.mjs:270
AboutHomeStartupCache: requestCache called with no _procManager! BrowserGlue.sys.mjs:6417:16`
